### ISSUE TITLE
Javascript callback - when passing a struct in as a parameter, the javascript function receives a null value.

### ISSUE THREAD

**Issue by tgroff-cargas** [cefsharp/CefSharp#1407]
I'm trying to return a simple struct to a javascript callback on a  method on a bound object.  However, in javascript, the parameter value always returns null.  When I pass in a string to the callback (like in the BoundObject.TestCallback method), everything works as expected.

Stepping through SerializeV8SimpleObject in V8Serialization.cpp, I see it getting the fieldNames for the properties on my object and setting those on subDict.  I don't follow how the parameter list gets updated to include subDict, however.  

Relevant code from SerializeV8SimpleObject;
```
                else if (type->IsValueType && !type->IsPrimitive && !type->IsEnum)
                {
                    auto fields = type->GetFields();
                    auto subDict = CefDictionaryValue::Create();
                    for (int i = 0; i < fields->Length; i++)
                    {
                        auto fieldName = fields[i]->Name;
                        auto strFieldName = StringUtils::ToNative(safe_cast<String^>(fieldName));
                        auto fieldVal = fields[i]->GetValue(obj);
                        SerializeV8SimpleObject(fieldVal, subDict, strFieldName, seen);
                    }
                } 
```

I've reproduced this issue on a fork of the CefSSharp.MinimalExample project that can be found here:

https://github.com/tgroff-cargas/CefSharp.MinimalExample

Thank you very much.

**Comment by amaitland** [comment]
Hopefully this is resolved in #1426, if you have time please take it for a spin. Thanks for the detailed bug report :+1:

**Comment by tgroff-cargas** [comment]
This is perfect – exactly what I needed.  Thank you very much!
